### PR TITLE
I picked blue as my color

### PR DESCRIPTION
**Pull request by kdaigle**
The color blue is great. You :ship: this now!

### CODE REVIEW

**Comment by emmapc** [comment]
:bird:

**Comment by zpainter** [comment]
You ain't got no lines to comment on.

**Comment by DanielleDionneAdams** [comment]
I think teal is more original...

**Comment by karensluzenski** [comment]
Blue is OK but green is better.

**Comment by skooog** [comment]
:+1:

**Comment by mentogithub** [comment]
:8ball:

**Comment by vmdowney** [comment]
I agree :ship: it!

File: colors/blue.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+I can't believe that:
+
+* It's blue.
+* Daboodeedaboda.
+* Yup, I did that.
+
+### Summary
+
+You should have chosen blue.
```
Comment (eslao) [inline colors/blue.md:13]: :earth_americas:

File: colors/blue.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Blue
+
+Blue is a great color because it's not red or green. It also is the color of awesome little LEDs. :lightbulb:
```
Comment (jmartinezbc) [inline colors/blue.md:3]: I agree :+1:

File: colors/blue.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+I can't believe that:
+
+* It's blue.
+* Daboodeedaboda.
+* Yup, I did that.
```
Comment (thurwitz) [inline colors/blue.md:9]: I think we all did

File: colors/blue.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Blue
```
Comment (emilyuna) [inline colors/blue.md:1]: What about green? :pear:

**Comment by brendanshort** [comment]
:shipit:

**Comment by lapalmer** [comment]
Purple is good too. :purple_heart:

**Comment by scorvene** [comment]
:camel:

**Comment by aabn** [comment]
I like purple too!

**Comment by mmwisner** [comment]
Little white lights are nice for the holidays.

**Comment by archdiva** [comment]
I can't keep up!!!!

**Comment by hmcmull** [comment]
Indigo, Sky, Cobalt or Navy?

**Comment by clementine11** [comment]
:yellow_heart:

**Comment by susabers** [comment]
:dromedary_camel: I like blue camels.

**Comment by thurwitz** [comment]
:+1:

**Pull request merged**